.github/workflows/windows.yml

```diff
@@ -0,0 +1,42 @@
+name: Windows Tests
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Is this supposed to be windows? And do we need two workflows for this? Couldn't we have a windows job in the normal ci workflow?